client/my-sites/stats/jetpack-upsell-section/upsell-card/available-upsells.tsx

```diff
@@ -0,0 +1,122 @@
+import {
+	PLAN_JETPACK_SECURITY_T1_YEARLY,
+	PRODUCT_JETPACK_BACKUP_T1_YEARLY,
+	PRODUCT_JETPACK_BOOST,
+	PRODUCT_JETPACK_SEARCH,
+	PRODUCT_JETPACK_SOCIAL_BASIC,
+	PRODUCT_JETPACK_VIDEOPRESS,
+} from '@automattic/calypso-products';
+import { translate } from 'i18n-calypso';
+import SecurityIcon from './icons/jetpack-icon-lock.svg';
+import BackupIcon from './icons/jetpack-product-icon-backup.svg';
+import BoostIcon from './icons/jetpack-product-icon-boost.svg';
+import SearchIcon from './icons/jetpack-product-icon-search.svg';
+import SocialIcon from './icons/jetpack-product-icon-social.svg';
+import VideoPressIcon from './icons/jetpack-product-icon-videopress.svg';
+
+export type Product = {
+	description: string;
+	href: string;
+	iconUrl: string;
+	isFree: boolean;
+	slug: string;
+	title: string;
+	features: string[];
+	checkoutSlug: string;
+	checkoutUrl: string | null;
+};
+
+export function getAvailableUpsells() {
+	return [
+		{
+			description: translate(
+				'Protect your site from hackers and spam with automated backups, malware scanning, and spam filtering.'
+			),
+			href: 'https://jetpack.com/features/security/',
+			iconUrl: SecurityIcon,
+			isFree: false,
+			slug: 'security',
+			title: translate( 'Security', { context: 'Jetpack product name' } ),
+			features: [ 'scan' ],
+			checkoutSlug: PLAN_JETPACK_SECURITY_T1_YEARLY,
+		},
+		{
+			description: translate(
+				'Save every single change and get back online quickly with one-click restores.'
+			),
+			href: 'https://jetpack.com/upgrade/backup/',
+			iconUrl: BackupIcon,
+			isFree: false,
+			slug: 'backup',
+			title: translate( 'Backup' ),
+			features: [ 'backups', 'restore' ],
```

Comment:
Backup is still showing for Complete plan. I don't find `restore` in active features. Would it be related?

Reply:
I'll give this another look this afternoon.

Reply:
I'm not sure where I found that previously but now, when adding a Backup plan, I'm seeing the following feature slugs:

- backups
- full-activity-log
- priority_support
- real-time-backups

I'll update the PR and test again. Thanks for the catch, Jasper! 👍

Reply:
Should be addressed in https://github.com/Automattic/wp-calypso/pull/99048/commits/91f8d172928a9fb1fdd89d12f4d1cebebaad3456.